src/Stripe.net/Infrastructure/Public/StripeClient.cs

```diff
@@ -1,23 +1,51 @@
 namespace Stripe
 {
+    using System;
     using System.Net;
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
     using Newtonsoft.Json.Linq;
+    using Stripe.Infrastructure;
 
     /// <summary>
     /// A Stripe client, used to issue requests to Stripe's API and deserialize responses.
     /// </summary>
     public class StripeClient : IStripeClient
     {
         /// <summary>Initializes a new instance of the <see cref="StripeClient"/> class.</summary>
+        /// <param name="apiKey">The API key used by the client to make requests.</param>
+        /// <param name="clientId">The client ID used by the client in OAuth requests.</param>
```

Comment:
I don't fully grasp this `clientId`. Reading the rest of the PR it seems to be the ca_123. But naming that variable `clientId` in the StripeClient class is quite confusing (at least to me). I kind of assumed you meant some unique client identifiers for that instance for some reason until I read more and it started being clear it did not mean that.

Food for thought as I don't feel strongly but wanted to mention this.

Reply:
Yeah, the word "client" is a bit overloaded here. I think the documentation clarifies that this is the OAuth client ID, but I could rename the argument to `oauthClientId` or something similar to help disambiguate.